packages/backend/src/models/ResourceViewItemModel.ts

```diff
@@ -0,0 +1,57 @@
+import { ResourceViewItemType, ResourceViewSpaceItem } from '@lightdash/common';
+import { Knex } from 'knex';
+
+type Dependencies = {
+    database: Knex;
+};
+
+export class ResourceViewItemModel {
+    database: Knex;
+
+    constructor(dependencies: Dependencies) {
+        this.database = dependencies.database;
+    }
+
+    async getSpacesByPinnedListUuid(
+        pinnedListUuid: string,
+    ): Promise<ResourceViewSpaceItem[]> {
+        const rows = await this.database.raw<Record<string, any>[]>(
+            `
+select
+    pl.project_uuid,
```

Comment:
should we also filter by project_uuid or org_uuid ?

Reply:
My thinking was to apply permissions after in the service. So get the item by uuid, and then check that the user has access to the project/org